0001-CVE-triage-of-some-libjpeg-CVEs.patch

```diff
@@ -0,0 +1,234 @@
+ 
+ Patches_libjpeg9:
+-upstream_libjpeg9: needs-triage
++upstream_libjpeg9: released (1:9c-1)
+ precise/esm_libjpeg9: DNE
+ trusty_libjpeg9: DNE
+ trusty/esm_libjpeg9: DNE
+-xenial_libjpeg9: needs-triage
++xenial_libjpeg9: needed
+ artful_libjpeg9: ignored (reached end-of-life)
+-bionic_libjpeg9: needs-triage
++bionic_libjpeg9: needed
+ cosmic_libjpeg9: ignored (reached end-of-life)
+ disco_libjpeg9: not-affected (1:9c-2)
+ eoan_libjpeg9: not-affected (1:9c-2)
+diff --git a/active/CVE-2018-11214 b/active/CVE-2018-11214
+index 7230d24ee4..85428e83da 100644
+--- a/active/CVE-2018-11214
++++ b/active/CVE-2018-11214
+@@ -24,7 +24,7 @@ CVSS:
+  nvd: CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H
+ 
+ Patches_libjpeg6b:
+-upstream_libjpeg6b: needs-triage
++upstream_libjpeg6b: deferred
+ precise/esm_libjpeg6b: DNE
+ trusty_libjpeg6b: ignored (out of standard support)
+ trusty/esm_libjpeg6b: needed
+@@ -40,7 +40,7 @@ devel_libjpeg6b: needed
+ 
+ Patches_libjpeg-turbo:
+  upstream: https://github.com/libjpeg-turbo/libjpeg-turbo/commit/6709e4a0cfa44d4f54ee8ad05753d4aa9260cb91
+-upstream_libjpeg-turbo: needs-triage
++upstream_libjpeg-turbo: released (1.3.0-0ubuntu2.1)
+ precise/esm_libjpeg-turbo: released (1.1.90+svn733-0ubuntu4.5)
+ trusty_libjpeg-turbo: released (1.3.0-0ubuntu2.1)
+ trusty/esm_libjpeg-turbo: released (1.3.0-0ubuntu2.1)
+@@ -55,13 +55,13 @@ groovy_libjpeg-turbo: not-affected (1.5.2-0ubuntu5)
+ devel_libjpeg-turbo: not-affected (1.5.2-0ubuntu5)
+ 
+ Patches_libjpeg9:
+-upstream_libjpeg9: needs-triage
++upstream_libjpeg9: released (1:9c-1)
+ precise/esm_libjpeg9: DNE
+ trusty_libjpeg9: DNE
+ trusty/esm_libjpeg9: DNE
+-xenial_libjpeg9: needs-triage
++xenial_libjpeg9: needed
+ artful_libjpeg9: ignored (reached end-of-life)
+-bionic_libjpeg9: needs-triage
++bionic_libjpeg9: needed
+ cosmic_libjpeg9: ignored (reached end-of-life)
+ disco_libjpeg9: not-affected (1:9c-2)
+ eoan_libjpeg9: not-affected (1:9c-2)
+diff --git a/active/CVE-2018-11813 b/active/CVE-2018-11813
+index e864d7c755..54b9477dbd 100644
+--- a/active/CVE-2018-11813
++++ b/active/CVE-2018-11813
+@@ -16,23 +16,23 @@ CVSS:
+  nvd: CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H
+ 
+ Patches_libjpeg6b:
+-upstream_libjpeg6b: needs-triage
++upstream_libjpeg6b: needed
+ precise/esm_libjpeg6b: DNE
+ trusty_libjpeg6b: ignored (out of standard support)
+-trusty/esm_libjpeg6b: needs-triage
+-xenial_libjpeg6b: needs-triage
+-bionic_libjpeg6b: needs-triage
++trusty/esm_libjpeg6b: needed
++xenial_libjpeg6b: needed
++bionic_libjpeg6b: needed
+ eoan_libjpeg6b: ignored (reached end-of-life)
+-focal_libjpeg6b: needs-triage
+-groovy_libjpeg6b: needs-triage
+-devel_libjpeg6b: needs-triage
++focal_libjpeg6b: needed
++groovy_libjpeg6b: needed
++devel_libjpeg6b: needed
+ 
+ Patches_libjpeg-turbo:
+  upstream: https://github.com/libjpeg-turbo/libjpeg-turbo/commit/909a8cfc7bca9b2e6707425bdb74da997e8fa499
+-upstream_libjpeg-turbo: needs-triage
++upstream_libjpeg-turbo: released (2.0.3-0)
```

Comment:
Based on https://security-tracker.debian.org/tracker/CVE-2018-11813, it looks like t his was fixed in 1:2.0.5-1